data_structures/binary_tree/maximum_sum_bst.py

```diff
@@ -0,0 +1,68 @@
+class TreeNode:
```

Comment:
Please make this a https://docs.python.org/3/library/dataclasses.html

Reply:
See https://github.com/TheAlgorithms/Python/blob/master/data_structures/binary_tree/binary_tree_traversals.py#L10 for a working example.